third_party/lmdb.BUILD

```diff
@@ -18,8 +18,20 @@ cc_library(
     copts = [
         "-w",
     ],
-    linkopts = [
-        "-lpthread",
-    ],
+    linkopts = select({
+        ":windows": ["-Wl,advapi32.lib"], # InitializeSecurityDescriptor, SetSecurityDescriptorDacl
+        ":windows_msvc": ["-Wl,advapi32.lib"],
+        "//conditions:default": ["-lpthread"],
+    }),
     visibility = ["//visibility:public"],
 )
+
+config_setting(
+    name = "windows",
+    values = {"cpu": "x64_windows"},
```

Comment:
I'm soon going to send a change to make TF build on Windows without `--cpu=x64_windows_msvc`. So here also adding config_setting for cpu value x64_windows to be compatible.